install/source/dependencies/index.markdown

```diff
@@ -73,6 +73,17 @@ Next manually add a package.xml as used in the ROS release wrapper for OMPL (mod
 
 Now you should be able to build using regular ``catkin build``
 
+If you are building packages that use your source install of OMPL, you should not use ``catkin_package(CATKIN_DEPENDS ompl)`` to link agianst your source install. Instead you should include OMPL just like you would include Boost or Eigen.
```

Comment:
This is an alternative approach to the first section (that I wrote) about using catkin to build ompl. Separate the two using triple hash tag sections (###)

Reply:
It is plain *wrong* to add ompl in `CATKIN_DEPENDS`!
OMPL is a standalone package that can be built in a catkin-tools workspace, it is *still no catkin package*.
I'm not sure I ever needed to do this before, but assuming your headers export OMPL includes, or your libraries need to be linked together with OMPL, then you should add `catkin_package(DEPENDS OMPL)`.